File: 프로그래머스/0/120891. 369게임/369게임.js
function solution(order) {
    let orderStr = order.toString();
    
    return Array.from({length: orderStr.length}, (_, i) => {
                    return +orderStr.charAt(i)
                })
                .filter(e => e != 0 && e % 3 == 0)
                .length;
}

File: 프로그래머스/0/120891. 369게임/369게임.py
def solution(order):
    orderStr = str(order)
    orderIntArr = map(lambda x: int(orderStr[x]), range(len(orderStr)))
    
    return len(list(filter(is_369, orderIntArr)))

def is_369(num):
    return num != 0 and num % 3 == 0